lib/susedistribution.pm

```diff
@@ -168,5 +166,55 @@ sub become_root() {
     send_key('ctrl-l');
 }
 
+# initialize the consoles needed during our tests
+sub activate_consoles {
+    my ($self) = @_;
+    if (get_var('BACKEND', 'qemu')) {
+        activate_console('install-shell',    'tty-console', {tty => 2});
+        activate_console('root-console',     'tty-console', {tty => 2});
+        activate_console('bernhard-console', 'tty-console', {tty => 4});
```

Comment:
I was thinking something like "select_console 'console', 'root'" or "select_console 'console', user => 'bernhard'".
But whatever, at first it felt strange to select console by one part of console name and then select user by the another. But as more I think about it ,the more I'm fine with it. Too high order for me apparently :)